islands/mint/MintContent.tsx

```diff
@@ -0,0 +1,769 @@
+        />
+      </div>
+
+      <div class="w-full">
+        <p class="text-lg font-semibold text-[#F5F5F5] mb-3">
+          Repeat Mint
+        </p>
+        <input
+          type="text"
+          class="px-3 py-6 bg-[#6E6E6E] text-sm text-[#F5F5F5] w-full"
+          placeholder="Number"
+        />
+      </div>
+
+      {
+        /* <div
+        class={"bg-gray-800 w-[340px] px-4 py-4 rounded-lg"}
+        style={{ border: "solid 1px gray" }}
+      >
+        <span class={"text-white text-[16px]"}>Optimization</span>
+        <div class="flex flex-col mb-4 gap-4 pt-2">
+          <div class={"flex items-center"}>
+            <input
+              id="default-radio-1"
+              type="radio"
+              name="radio"
+              class="w-4 h-4 focus:ring-blue-500 focus:ring-2"
+            />
+            <label
+              for="default-radio-1"
+              class="ms-2 text-sm font-medium text-white"
+            >
+              None
+            </label>
+          </div>
+          <div class={"flex items-center"}>
+            <input
+              id="default-radio-2"
+              type="radio"
+              name="radio"
+              class="w-4 h-4 focus:ring-blue-500 focus:ring-2"
+            />
+            <label
+              for="default-radio-2"
+              class="ms-2 text-sm font-medium text-white"
+            >
+              Max compression
+            </label>
+          </div>
+          <div class={"flex items-center"}>
+            <input
+              id="default-radio-2"
+              type="radio"
+              name="radio"
+              class="w-4 h-4 focus:ring-blue-500 focus:ring-2"
+            />
+            <label
+              for="default-radio-2"
+              class="ms-2 text-sm font-medium text-white"
+            >
+              Balanced
+            </label>
+          </div>
+          <div class={"flex items-center"}>
+            <input
+              id="default-radio-2"
+              type="radio"
+              name="radio"
+              class="w-4 h-4 focus:ring-blue-500 focus:ring-2"
+            />
+            <label
+              for="default-radio-2"
+              class="ms-2 text-sm font-medium text-white"
+            >
+              Max quality
+            </label>
+          </div>
+          <span class={"text-gray-400 pt-4"}>Note: There are no refunds</span>
+          <span class={"text-gray-400"}>
+            Only mint if the preview is of acceptable quality!
+          </span>
+          <span class={"text-gray-400"}>
+            Sometimes it's ok to not optimize!
+          </span>
+        </div>
+      </div> */
+      }
+
+      <div class={"w-full flex flex-col gap-2"}>
+        <div class="flex justify-between">
+          <span class={"text-[#F5F5F5]"}>
+            EFFECTIVE FEE RATE: {(fee / 10.0).toFixed(2)} sat/vB
+          </span>
+          <span class={"text-[#F5F5F5] hidden md:block"}>
+            RECOMMENDED: 78 sat/vB
```

Comment:
this should be a multiplier. something like 1.5x the current next block fee rate.

Reply:
got it.
I will change this.